__tests__/__snapshots__/LoadedApp.snapshot.tsx.snap

```diff
@@ -1,326 +1,3 @@
-              </Text>
-              <Text
-                onPress={[Function]}
-                style={
-                  {
-                    "color": "#c3c3c3",
-                    "fontSize": 14,
-                    "fontWeight": "600",
-                    "opacity": 1,
-                    "paddingTop": 25,
-                  }
-                }
-                testID="menu.walletseedufvk"
-              >
-                text translated
-              </Text>
-              <Text
-                onPress={[Function]}
-                style={
-                  {
-                    "color": "#c3c3c3",
-                    "fontSize": 14,
-                    "fontWeight": "600",
-                    "opacity": 1,
-                    "paddingTop": 25,
-                  }
-                }
-                testID="menu.insight"
-              >
-                text translated
-              </Text>
-              <Text
-                onPress={[Function]}
-                style={
-                  {
-                    "color": "#c3c3c3",
-                    "fontSize": 14,
-                    "fontWeight": "600",
-                    "opacity": 1,
-                    "paddingTop": 25,
-                  }
-                }
-                testID="menu.tipzingolabs"
-              >
-                text translated
-              </Text>
-              <Text
-                onPress={[Function]}
-                style={
-                  {
-                    "color": "#c3c3c3",
-                    "fontSize": 14,
-                    "fontWeight": "600",
-                    "opacity": 1,
-                    "paddingTop": 25,
-                  }
-                }
-              >
-                text translated
-              </Text>
-            </View>
-          </View>
-        </RCTScrollView>
-        <View
-          style={
-            {
-              "backgroundColor": "#040C17",
-              "bottom": 5,
-              "flexDirection": "row",
-              "padding": 10,
-              "position": "absolute",
-            }
-          }
-        >
-          <Text
-            style={
-              {
-                "color": "#ffffff",
-                "fontSize": 8,
-              }
-            }
-          >
-            Version : 
-          </Text>
-          <Text
-            style={
-              {
-                "color": "#5a8c5a",
-                "fontSize": 8,
-              }
-            }
-          >
-            text translated
-          </Text>
-          <Text
-            style={
-              {
-                "color": "#ffffff",
-                "fontSize": 8,
-                "marginLeft": 10,
-              }
-            }
-          >
-            text translatedtext translated
-          </Text>
-        </View>
-      </View>
-    </View>
-  </View>,
-  <View />,
-]
-`;
+exports[`Component LoadedApp - test LoadedApp - snapshot 1`] = `<View />`;
```

Comment:
This snapshot is empty, we need to find the way to mock the new drawer properly... I think `Screen` => null remove all the content.